Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: all
+all: install test
```

Comment:
This means that everyone calling "make" without arguments gets some non-package-maintained application installed into their work environment depending on how sudo is configured. That would be not so nice. I suggest to remove the target from all and call it in CI explicitly